tests/packages/test_locker.py

```diff
@@ -177,5 +177,47 @@ def test_lock_file_should_not_have_mixed_types(locker, root):
     with locker.lock.open(encoding="utf-8") as f:
         content = f.read()
 
-    print(content)
+    assert expected == content
+
+
+def test_lock_extra_duplicates_main(locker, root):
```

Comment:
I think this test duplicates the one above, doesn't it?

Reply:
Now we have three related tests:
 - `test_lock_file_should_not_have_mixed_types`
 - `test_lock_extra_duplicates_main`
 - `test_add_extra_duplicates_main`

All the tests are similar in general, so please let me know which ones should be removed/modified.
Thanks!

Reply:
I think for the sake of simplicity we can keep `test_lock_file_should_not_have_mixed_types` only.

Reply:
I've removed two other tests.